bot/cogs/template.py

```diff
@@ -0,0 +1,14 @@
+from discord.ext import commands
+
+
+class Template:
+    def __init__(self, bot):
+        self.bot = bot
+        
+    @commands.command()
```

Comment:
Perhaps we could use command groups in the template, and perhaps we could make use of the optional parameters in the decorator so that we can demonstrate to our future contributors that it is possible to give them names and group them together?

Reply:
Now contains examples of command groups and commands.